examples/resources/incident_catalog_entry/resource.tf

```diff
@@ -32,6 +32,8 @@ resource "incident_catalog_entry" "service_tier" {
 
   name = each.value.name
 
+  external_id = each.value.name
```

Comment:
I think this is a little confusing and we should make the example better with a distinct name and id